src/sensor.cpp

```diff
@@ -471,6 +471,19 @@ namespace librealsense
         else if (!_is_opened)
             throw wrong_api_call_sequence_exception("start_streaming(...) failed. UVC device was not opened!");
 
+        // RS2_OPTION_AUTO_EXPOSURE_PRIORITY will be disabled by default in firmware
+        // temporarily disable here to facilitate android stability tests
```

Comment:
To facilitate stability tests? Is this not meant to get into the code?

Reply:
removed.